packages/core/src/components/models/fieldGroups/ModelFieldGroupReference.js

```diff
@@ -56,12 +61,16 @@ export const ModelFieldReferenceBaseInput = ({ model, ...props }) => {
   });
 
   const selectedOption = useMemo(() => {
+    if (parent && refModel.model === parentModel.model) {
+      return results?.filter((e) => `${e.id}` === `${parent.id}`);
```

Comment:
This will show the default in the UI, but will it actually set the value in RHF?